libs/go/libp2p_node/aea/api.go

```diff
@@ -71,7 +65,7 @@ type AeaApi struct {
 	msgin_path      string
```

Comment:
naming convention should be fixed here, but I think better to do it in future PRs